app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post, foreign_key: 'postId'
+  belongs_to :user, foreign_key: 'authorId'
+
+  def update_comments_counter
+    post.update(commentsCounter: post.comments.count)
+  end
```

Comment:
- [ ] I think it would be better if you use the same approaches I mentioned in the previous comment, as it is the same logic. You should improve the `update_comments_counter` so  it update the comments counter for a `User` when he has added a new comment. For that I suggest you use the `increment!` method as the current one is set the attribute to be the count of associated comments, instead of incrementing the attribute by 1 after creating a new like.

- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_posts_counter` method every time a new post has been saved to database, then make the `update_comments_counter` method privateStudents can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the `update_comments_counter` method as `private`.
e.g.:
```ruby
class Comment < ApplicationRecord

   ...
  after_save :update_comments_counter

  private

  def update_comments_counter
    post.incremrnt!(:commentsCounter)
  end
end
```